app/Main.hs

```diff
@@ -0,0 +1,369 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE RankNTypes #-}
+
+module Main where
+
+import Prelude
+
+import Cardano.Binary
+    ( FromCBOR (..), ToCBOR (..) )
+import Cardano.Crypto.ProtocolMagic
+    ( ProtocolMagicId (..) )
+import Control.Exception
+    ( bracket )
+import Control.Monad
+    ( guard, void )
+import Data.ByteArray.Encoding
+    ( Base (..), convertFromBase, convertToBase )
+import Data.ByteString
+    ( ByteString )
+import Data.Maybe
+    ( isNothing )
+import Data.Text
+    ( Text )
+import Data.UTxO.Transaction
+    ( MkPayment (..) )
+import Data.UTxO.Transaction.Cardano.Byron
+    ( Byron
+    , decodeCoinSel
+    , decodeTx
+    , encodeCoinSel
+    , encodeTx
+    , fromBase16
+    , fromBase58
+    , mainnetMagic
+    , mkInit
+    , mkInput
+    , mkOutput
+    , mkSignKey
+    , testnetMagic
+    )
+import Data.Word
+    ( Word32 )
+import Numeric.Natural
+    ( Natural )
+import Options.Applicative.Help.Pretty
+    ( hardline, indent, string, vsep )
+import System.Console.ANSI
+    ( Color (..)
+    , ColorIntensity (..)
+    , ConsoleLayer (..)
+    , SGR (..)
+    , hSetSGR
+    , hSupportsANSIWithoutEmulation
+    )
+import System.Exit
+    ( exitFailure )
+import System.IO
+    ( Handle, hIsTerminalDevice, stderr, stdin, stdout )
+
+import Options.Applicative
```

Comment:
here no left space needed plus it would be good to comply with our coding standards for imports